rts/Sim/Path/TKPFS/PathEstimator.cpp

```diff
@@ -77,6 +78,82 @@ void CPathEstimator::AddCache(const IPath::Path* path, const IPath::SearchResult
 	pathingState->AddCache(path, result, strtBlock, goalBlock, goalRadius, pathType, synced);
 }
 
+bool CPathEstimator::SetStartBlock(
+	const MoveDef& moveDef,
+	const CPathFinderDef& peDef,
+	const CSolidObject* owner,
+	float3 startPos
+)
+{
+	constexpr size_t maxBlocksToCheck = 9;
```

Comment:
why 9? what happens if I change it to 42? consider:
```cpp
/* Increasing the value tradess off X for Y.
 * The value of 9 chosen because of Z. */
constexpr size_t maxBlocksToCheck = 9;
```